### PR TITLE
data: streetstylecoder

### PR DESCRIPTION
**Pull request by streetstylecoder**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5939"></a>

### CODE REVIEW

**Review by tangorishi — DISMISSED** [review]
@streetstylecoder good going! 👍🏻

File: data/streetstylecoder.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name":"Harsh Dhariwal",
+  "displayStatsPublic":"true",
```
Comment (kumarsonsoff3) [inline data/streetstylecoder.json:3]: We no longer need this `displayStatsPublic` key. So please, remove this from your `json`

```suggestion
```

File: data/streetstylecoder.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "type":"personal",
+  "bio":"backend developer and a somphomore at JIIT Noida",
+  "avatar":"https://avatars.githubusercontent.com/u/89967721?v=4",
```
Comment (kumarsonsoff3) [inline data/streetstylecoder.json:6]: Also, this one is no longer needed. 

```suggestion
```

**Review by kumarsonsoff3 — DISMISSED** [review]
Hey @streetstylecoder, please add a few links in the `links` section. You might want to look at the quickstart guide to help you with this: https://linkfree.io/docs/quickstart

Also, make the changes requested below. 🛠

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**